Products/ZenModel/migrate/ChangeConInfoEngine.py

```diff
@@ -0,0 +1,69 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2015, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import Migrate
+import logging
+import sys
+import os
+from subprocess import Popen, PIPE
+from Products.ZenUtils import GlobalConfig
+
+log = logging.getLogger('zen.Migrate')
+
+class ChangeConInfoEngine(Migrate.Step):
+    " Change engine for connection_info table. "
+
+    version = Migrate.Version(4, 2, 4)
```

Comment:
The version of the migrate script needs to be updated to 5.0.70 since this is develop